lib/jekyll-remote-theme.rb

```diff
@@ -19,7 +19,10 @@ class DownloadError < StandardError; end
     autoload :Theme,       "jekyll-remote-theme/theme"
     autoload :VERSION,     "jekyll-remote-theme/version"
 
-    CONFIG_KEY  = "remote_theme"
+    CONFIG_REPOSITORY_KEY = "repository"
+    CONFIG_THEME_KEY      = "remote_theme"
+    CONFIG_HEADERS_KEY    = "remote_headers"
```

Comment:
Why would a user need to be able to specify custom headers?

Reply:
To allow downloading a private hosted theme and additional headers required in a large organization.